### PR TITLE
Dm assessment in namely documentation

### PR DESCRIPTION
**Pull request by desi**
Document doing assessments using Namely.

### CODE REVIEW

File: markdown/assessment.md [hunk 1]
```diff
@@ -1,13 +1,52 @@
...
 
-This assessment will be used to make the decision about whether the apprentice shall join thoughtbot at the end of
-their apprenticeship, and to constructively work on any issues with the apprentice.
+To setup the assessment, go to [the Namely Review Creation page][reviews].
 
-The assessment you write should not be a surprise to your apprentice. You should be meeting with your apprentice on
-a weekly basis to communicate these points to them.
+## Conduct an Assessment
+
+* Choose A Reviewee: Select the apprentice getting reviewed
```
Comment (getaaron) [inline markdown/assessment.md]: - Some of these bullet points end in a period, but others don't
- The capitalization in these list items appears random/arbitrary. Consider just using normal sentence case, or if you want to use title capitalization for some reason you can run it through http://titlecapitalization.com/ to check AP style guidelines. For example, "Allow Reviewers To Opt Out Of Review" could be changed to "Allow Reviewers to Opt out of Review".

File: markdown/assessment.md [hunk 2]
```diff
@@ -1,13 +1,52 @@
...
+* Select Reviewers: Enter the name of the mentor.
+* Template For Reviewers: Choose "Apprentice".
+* Allow Reviewers To Opt Out Of Review: **Unchecked**
+* Select Review Administrators: Enter yourself. Namely only allows one
+  administrator currently.
+* Due Date: Set a reasonable due date in the future.
+
+## Reminder Options
+
+* Set a reasonable reminder date. Perhaps for the investment day so that the
```
Comment (getaaron) [inline markdown/assessment.md]: - The second sentence here is a sentence fragment (no active verb). One simple fix would be to change "for" to "use"

File: markdown/assessment.md [hunk 3]
```diff
@@ -1,13 +1,52 @@
...
+  mentor has time to respond.
+
+## Permission Options
+
+* Hide Reviewers Identities From One Another: Unchecked
```
Comment (getaaron) [inline markdown/assessment.md]: I think you need an apostrophe here: *Reviewers' Identities*

**Comment by getaaron** [comment]
I added some minor grammatical suggestions. I'm not sure this is as helpful for readers outside of thoughtbot as the old version though, since they might not be using Namely.

Also this document is almost completely procedural, whereas the doc it replaces has some guidelines on writing good assessments. Much of the advice that's deleted is good. Maybe consider keeping the old version, and putting the Namely instructions in a new file?

**Comment by cpytel** [comment]
@desi I think it makes sense to keep the guidelines and questions at the top and the instructions below. In the Handbook we do a similar thing, where the questions are actually there, that way we can also make suggestions to changing them through a PR process.

**Comment by desi** [comment]
@cpytel Can you take another quick look at this? I put the previous questions at the top and the review instructions below. I wasn't sure if your comment previously meant you wanted me to leave these questions here and move the actual review instructions into the handbook and link from here or do what I have done in the PR currently. Happy to move to handbook if that is preferred.

**Comment by cpytel** [comment]
Yes, what you’ve done here is what I meant. I think this looks good.

> On Jan 27, 2016, at 11:00 PM, Desi McAdam <notifications@github.com> wrote:
> 
> @cpytel <https://github.com/cpytel> Can you take another quick look at this? I put the previous questions at the top and the review instructions below. I wasn't sure if your comment previously meant you wanted me to leave these questions here and move the actual review instructions into the handbook and link from here or do what I have done in the PR currently. Happy to move to handbook if that is preferred.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/apprenticeship/pull/24#issuecomment-175903140>.
> 

---
Chad Pytel, Founder and CEO
617-482-1300 x113
http://thoughtbot.com 
http://twitter.com/thoughtbot

**Pull request merged**